Document/0x05b-Basic-Security_Testing.md

```diff
@@ -8,7 +8,7 @@ This section will give an overview of different methods on how an Android app ca
 
 #### Preparation
 
-The goal of a test is to verify if the app and the endpoint(s) it's communicating with, are implemented in a secure way. Several security controls like SSL Pinning or root detection might be implemented. These can slow down the testing dramatically and might already take days to bypass, depending on the implementation.
+The goal of a test is to verify if the app and the client side of it's communication with the endpoint(s), are implemented in a secure way. Several security controls like SSL Pinning or root detection might be implemented. These can slow down the testing dramatically and might already take days to bypass, depending on the implementation.
```

Comment:
> The goal of a test is to verify if the app and the endpoint(s) it's communicating with, are implemented in a secure way. 

it's -> its, and remove the comma after "with"

also "verify if the app and the client side of its communication with the endpoint(s)" is a weird-sounding construction. I'd rather write "verify if network communication is implemented securely on the client".